packages/ui-predicate-react/src/ui-predicate.css

```diff
@@ -0,0 +1,12 @@
+.ui-predicate__main {
```

Comment:
we might want to share this CSS between react & vuejs (as well as other framework) right?

Reply:
This CSS I might want to share yes.
So far I haven't had any luck including the CSS correctly when using the package. I need to look into it. 

Where do you think we should place the css in this case?
And do you think we should do this in this PR, or open up a second one for it?

Reply:
Indeed, we can do this later don't worry :)